src/aind_metadata_service/sharepoint/client.py

```diff
@@ -93,7 +96,7 @@ def get_procedure_info(
     @staticmethod
     def _handle_response_from_sharepoint(
         subject_id: str, subject_procedures=None
-    ) -> JSONResponse:
+    ) -> Tuple[int, Union[BaseModel, None]]:
```

Comment:
Instead of Union[BaseModel, None], it's more customary to use Optional[BaseModel]